### DISCUSSION TITLE
Sort by upvotes

### DISCUSSION THREAD

**Discussion by 3kh0** [community/community#10585] [Discussions] [answered]
I think it would be a very cool idea to add a sort function to sort discussions by upvotes, just like how in Reddit you can sort by highest number of upvotes.

**Comment by markseuffert** [comment, marked as answer]
See also #3328

**Comment by TPS** [comment]
@3kh0 Duplicate https://github.com/orgs/community/discussions/3328 is marked <kbd>Shipped</kbd>.  Close FTW?
